.changelog/4483.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Introduces the `marketplaces` new field in the breaking changes release notes ".json" file. This supports choosing on which marketplace the braking changes message will appear.
```

Comment:
add that this field is not require and without it it will added to all marketplaces